src/components/ArtistProfiles.res

```diff
@@ -171,7 +171,9 @@ module ArtistDetails = {
           }}
         </Rimble.Box>
         <Rimble.Box p=1 width=[1., 1., 0.3333]>
-          <h2> {"Total contributed by artist"->React.string} </h2>
+          <h2> {"Total contributed through artwork created by "->React.string} {optName
+                ->Option.getWithDefault(Option.getWithDefault(optArtistName, "Loading artist name"))
```

Comment:
It is a bit funny that it would say "Total contributed through artwork created by Loading artist name" when the page loads. I think you should change the `getWithDefault` to a `mapWithDefault` and only if the user's name is defined should you use it. (same on line 187)

Reply:
general idea: 
```rescript
optArtistName->mapWithDefault("String to put before artist name is loaded", artistName => "string with " ++ artistName)
```